README.md

```diff
@@ -154,7 +154,7 @@ Get the source from github with:
     virtualenv venv
     . venv/bin/activate
     pip install -r requirements.txt
-    python -m charmcraft
+    CHARMCRAFT_DEVLOPER=1 python -m charmcraft
```

Comment:
DEVLOPER should be DEVELOPER

Reply:
also, what does this do?

Reply:
Without this, checking out the git repo and following `python -m charmcraft` immediately dies with an error log (see https://github.com/canonical/charmcraft/blob/342a2ec36c90aa6c7f199c9470256c6e18e760e7/charmcraft/env.py#L54). Seeing it permanently causes the testsuite to fail, though.